nf_core/download.py

```diff
@@ -13,6 +13,7 @@
 import tarfile
 import textwrap
 from datetime import datetime
+from typing import Any
```

Comment:
No. 

Although I promised to type the whole download command at some point, I have no intentions of doing this now. Copy/paste accident, because I copied a typed function as skeleton into the code, and VS Code happily helped me out with this auto-import.